chill-scala/src/main/scala/com/twitter/chill/ScalaKryoInstantiator.scala

```diff
@@ -252,5 +252,8 @@ class AllScalaRegistrar extends IKryoRegistrar {
     k.forClass[scala.runtime.VolatileByteRef](new VolatileByteRefSerializer)
     k.forClass[BigDecimal](new BigDecimalSerializer)
     k.register(Queue.empty[Any].getClass)
+    k.register(Map(1 -> 2).filterKeys(_ != 2).getClass)
+    k.register(Map(1 -> 2).mapValues(_ + 1).getClass)
+    k.register(Map(1 -> 2).keySet.getClass)
```

Comment:
I think we should use `TraversableSerializer` here. I think `.forConcreteTraversableClass` will do the job.